SmartDeviceLink/SDLResponseDispatcher.m

```diff
@@ -140,6 +141,17 @@ - (void)clear {
     _audioPassThruHandler = nil;
 }
 
+- (void)sdl_removeOldButtonHandlers {
+    for(SDLRPCCorrelationId *key in self.rpcRequestDictionary.allKeys) {
```

Comment:
This appears to remove all old buttons on every new `Show` request, but this will cause very bad behavior. For example:

* Show 1
text1, text2, 3 soft buttons

* Show 2
update text 1

Now all 3 soft buttons no longer work even though they're still displayed on the screen.